public/app/features/query/components/QueryGroup.tsx

```diff
@@ -8,12 +8,13 @@ import {
   DataSourceApi,
   DataSourceInstanceSettings,
   getDefaultTimeRange,
+  GrafanaTheme2,
   LoadingState,
   PanelData,
 } from '@grafana/data';
 import { selectors } from '@grafana/e2e-selectors';
 import { DataSourcePicker, getDataSourceSrv } from '@grafana/runtime';
-import { Button, CustomScrollbar, HorizontalGroup, InlineFormLabel, Modal, stylesFactory } from '@grafana/ui';
+import { Button, CustomScrollbar, HorizontalGroup, InlineFormLabel, Modal, Themeable2, withTheme2 } from '@grafana/ui';
```

Comment:
We can revert this file. These changes are not necessary for this feature.

Reply:
I think it's still a good change. To move it to the new theme system. Should we move it to another PR?

Reply:
Yes I think we should do that.